test/TestTokens.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.11;
+
+import "truffle/Assert.sol";
+import "truffle/DeployedAddresses.sol";
+import "../contracts/tokens/PrintableToken.sol";
+
+contract TestTokens {
+    function testPrintableToken() {
+        uint256 testPrintValue = 100;
+        PrintableToken testToken = new PrintableToken("testTokenName", 0, "testTokenSymbol", testPrintValue);
+        Assert.equal(testToken.printValue(), testPrintValue, "It should initialize printValue");
```

Comment:
Insert some whitespace between different steps of tests. Also, in general, add comments on what you're testing.

Reply:
(TestCommon is much less of an eyesore)

Reply:
Isn't it enough?
``` 
Assert.equal(expected, actual, "It should explain what this test should do");
```

Reply:
The comments in the assertions describe what you're expecting to happen, the comments in the code should explain what actions the test code takes (especially regarding creating mocks/fakes).